tasks/angular/NgRX.md

```diff
@@ -89,14 +88,14 @@ Maximum points - **100**
 - [ ] Saving data received from the YouTube API is implemented using NgRX library (**+30**)
 
 Fines
-- [ ] Failure to submit on time may lead to points lose according to the [Stage #2 requirements](https://docs.rs.school/#/stage2?id=%d0%94%d0%b5%d0%b4%d0%bb%d0%b0%d0%b9%d0%bd%d1%8b)
+- [ ] Failure to submit on time may lead to points lose according to the [Deadlines for Students requirements](https://docs.app.rs.school/#/platform/pull-request-review-process?id=deadlines-for-students)
 - [ ] The app has wrong project structure (**-20**)
 - [ ] The app doesn't work or has console errors (**-20**)
-- [ ] TSLint warnings or errors are present (**-15**)
+- [ ] ESLint warnings or errors are present (**-15**)
 
 ### Useful links
-https://ngrx.io  
-https://ngrx.io/guide/store/configuration/runtime-checks  
-https://ngrx.io/guide/store/actions  
-https://ngrx.io/guide/store/reducers  
-https://ngrx.io/guide/effects  
```

Comment:
These double whitespaces at the end of the line are necessary for linebreaks, please re-check correctness of your fix display

Reply:
I'm sorry for such a delay, I have read you message and forgot to make correctness. I have added whitespaces back